src/OmniSharp/AspNet5/AspNet5ProjectSystem.cs

```diff
@@ -416,7 +416,7 @@ private void TriggerDependeees(string path)
             }
         }
 
-        private void Initialize()
+        private void Startup()
```

Comment:
@davidfowl I don't know more than you... There was a public Initalize() (without the 'i') method on the Interface which I renamed Initialize() (as it should be, IMHO). But there was already a private method with that name so I renamed it too (the Startup one).

What name would you like me to use ? Do you want me to rollback these changes and let the Initalize() method like that ?